Bottom: 8b7c5fb27474fda3e349ca01c559179cc4a573b3
Top:    014baf6cffa98842a534c53aab78e90d2514512f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-24 21:17:04 -0400

Refresh of dma-module

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index 0d56849..c530dc7 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -1,5 +1,5 @@
 config SND_SOC_OF_SIMPLE
-	tristate
+	bool
 	
 config SND_MPC52xx_DMA
 	tristate
